app/components/optimized-power-select/options/component.js

```diff
@@ -0,0 +1,111 @@
+import { action } from "@ember/object";
```

Comment:
*documenting this since the handover is still quite fresh*
As @aziiee  told me: 
> The `OptimizedPowerSelect` is a new customized `ember-power-select` with less customizations than our other custom `PowerSelect` component, which will override less defaults and therefor should be easier to maintain. 

I will have a look at this in detail soon.